backend/internal/handlers/venues/venues.go

```diff
@@ -0,0 +1,40 @@
+package venues
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/google/uuid"
+)
+
+func (s *Service) DeleteVenue(c *fiber.Ctx) error {
+	fmt.Println("Pinging correct service")
+
+	venueID := c.Params("venueId")
+	if venueID == "" {
+		c.Status(http.StatusBadRequest)
```

Comment:
for consistency, use the err helper functions from [`errs`](https://github.com/GenerateNU/nightlife/blob/main/backend/internal/errs/api_err.go). if there isn't one that suits your needs, create a new one!